src/state/login/reducers.js

```diff
@@ -0,0 +1,21 @@
+import { ADD_EMAIL, ADD_PASSWORD } from './actions';
+
+const initialState = {
```

Comment:
I'd love to see `initialState.js` be it's own file for the same reason I wanted to see `src/state/index.js` become a thing. Developers would benefit from being able to see what variables exist in global state.